<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Random Dice</string>
  </property>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Dice</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="player2_text">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>650</y>
     <width>350</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_10">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_13">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_12">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_08">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_07">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_06">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_03">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_04">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_02">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_01">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_14">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_15">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_17">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_16">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="player1_text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>650</y>
     <width>350</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/player1_resize.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>260</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/player2_resize.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/player1_resize.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="player2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>510</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/player2_resize.png</pixmap>
   </property>
  </widget>
  <widget class="QTextBrowser" name="main_text">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>200</width>
     <height>90</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rule">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Rule</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/desert_island.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/storm.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/wallpapers.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/start_line.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
  <zorder>btn1</zorder>
  <zorder>player2_text</zorder>
  <zorder>land_10</zorder>
  <zorder>land_13</zorder>
  <zorder>land_11</zorder>
  <zorder>land_12</zorder>
  <zorder>land_08</zorder>
  <zorder>land_07</zorder>
  <zorder>land_06</zorder>
  <zorder>land_03</zorder>
  <zorder>land_04</zorder>
  <zorder>land_02</zorder>
  <zorder>land_01</zorder>
  <zorder>land_14</zorder>
  <zorder>land_15</zorder>
  <zorder>land_17</zorder>
  <zorder>land_16</zorder>
  <zorder>player1_text</zorder>
  <zorder>label_1</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>player1</zorder>
  <zorder>player2</zorder>
  <zorder>main_text</zorder>
  <zorder>rule</zorder>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
